src/WebUI/Pods/PodsLeftPanel.scss

```diff
@@ -1,10 +1,17 @@
-// todo :: remove once we have backbutton feature
-.pod-left-panel-header .pod-left-panel-back-button {
-    cursor: pointer;
-    margin-right: 4px;
+.pods-left-panel-row:not(.first-row) .bolt-list-cell {
+    // This is required as we are doing a custom implementation of list row, and the border is not set by default
+    border-top: 1px solid rgba(0,0,0,.08);
```

Comment:
if we have any variable for this please use. i do not find any variable for this, but we should check how they are getting this color in other places and should use that variable

Reply:
fixed